pages/samvera/developer_community/pr_checklist.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "PR Checklist"
+a-z: ["Pull Request", "PR", "PR Checklist", "Pull Request Checklist", "PR Review", "PR Creation" ]
+keywords: PR, Pull Request
+categories: Procedures
+permalink: pr-checklist.html
+folder: samvera/procedures/pr_checklist.md
+sidebar: home_sidebar
+tags: [procedures]
+---
+
+Checklist to use when creating and reviewing a PR.
+
+- Legal
+  - [ ] CLA:  The contributor has a valid CLA on file.
+  - [ ] Reviewer: At least one of the reviewers is from an institution other than the contributor's institution.
+- Code quality
+  - [ ] Code: Changes seem reasonable and meet the expected behavior stated in the related issue.  Changes follow best practices for the community.
+  - [ ] UI changes: Ad hoc testing of the expected behavior in the UI appears correct.  Expected behavior should be described in the related issue.
+  - [ ] UI existing: Ad hoc testing of related features in the UI appear to continue to operate correctly.
+  - [ ] Hound issues:  Hound issues are either fixed or comments identify why it doesn't need to be fixed.
+  - [ ] Test: There are tests written for all changes and new functionality.
+  - [ ] Test efficiency: Tests use factorybot `build` instead of `create` where ever possible.  This is especially important for repository objects that cause the object to be created in Fedora.
+  - [ ] Test coverage: Test coverage stayed the same or rose.  Review coveralls report for the PR.
```

Comment:
could we replace this with something about adding unit tests that ensure correct behavior of new classes/modules/methods? and then I would add some guidance about feature tests (but that may be harder agree on).